go/go-sql/.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
```

Comment:
Please delete this file.

Reply:
Can you add a .gitignore in the parent folder "go" to ignore such IDE generated files ?

Reply:
Added to **/.vscode/** to root gitignore